packages/arb-bridge-eth/contracts/arch/Hashing.sol

```diff
@@ -129,7 +129,7 @@ library Hashing {
         } else if (val.typeCode == Value.hashOnlyTypeCode()) {
             return bytes32(val.intVal);
         } else if (val.typeCode == Value.bufferTypeCode()) {
-            return keccak256(abi.encodePacked(uint256(123), val.bufferHash));
+            return keccak256(abi.encodePacked(val.size, val.bufferHash));
```

Comment:
This is inconsistent with the hashing done elsewhere in this PR.  The approach here seems like the correct one.